test/ui-test/testSuites/suite_messaging/tst_ChatFlow/bdd_hooks.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# This file contains hook functions to run as the .feature file is executed
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../../../testSuites/global_shared/"))
+sys.path.append(os.path.join(os.path.dirname(__file__), "../../../src/"))
+
+from steps.startupSteps import StartupSteps
+
+# Global properties for the specific feature
+_user = "tester123"
+_password = "TesTEr16843/!@00"
+_chat_room = "test"
+
+@OnFeatureStart
```

Comment:
Yeah.. I know.. but:
- The login process takes a long time so we need to reduce it by just doing the logins once and the way to do that is by adding it here.
- We can add as a backround step the navigation to the room but it has been also moved to here to reduce the execution time. We can consider this specific feature only for this specific room and if we need to interact between different public chats, then it will need a different test case. WDYT?